devtools/pip-tools/README.md

```diff
@@ -0,0 +1,49 @@
+# Bareos Source Sanity Checker
+
+This repository contains a pluggable python-based source sanity checker targeted to check all of the Bareos sources.
+
+The program selects files in your source directory to look at based on their git status (i.e. committed, changed, untracked). For each file all the plugins will be run and will check if the file is ok and - if possible and enabled - show what needs to be changed or apply the changes automatically.
+
+## Installation
+The program was developed and tested with python 3.6, so this is the minimum required version (we cannot describe this in the Pipfile).
+First of all, you need the virtualenv for the project. To install all required dependencies, just all `pipenv sync` in the top directory.
+After that can run included software with symlinks pointing to the supplied `pipenv-wrapper.sh`. If you add a symlink `check-sources` from one of your binary dirs (e.g. ~/.local/bin) to that script you'll have it available in your path.
```

Comment:
I think we should consequently change check-sources to bareos-check-sources everywhere.
I think the .check-sources-ignore filenames and special tags in files also should be renamed to bareos-check-sources.